tests/integration/policy/testdata/enable_envoy_ratelimit.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Do we want this as a sample? Or will we move it in the future

Reply:
Can add this as a sample.. so that it's easier when we add the documentation..

Reply:
Done